spec/controllers/collections_controller_spec.rb

```diff
@@ -25,12 +25,20 @@
   end
 
   describe 'GET #show' do
-    let(:collection) { create(:collection, valid_attributes) }
+    render_views
```

Comment:
I would suggest adding a new line after `render_views`
```suggestion
    render_views
    
```